src/actions/content/content.js

```diff
@@ -13,6 +13,7 @@ import {
   UPDATECOLUMNS_CONTENT,
   LOCK_CONTENT,
   UNLOCK_CONTENT,
+  FC_DELETE,
```

Comment:
@danielamormocea Maybe we can find a better name, more self-explenatory, for this action.